weave-js/src/components/PagePanelComponents/PersistenceManager.tsx

```diff
@@ -617,6 +619,32 @@ const HeaderFileControls: React.FC<{
   );
 };
 
+function useFetchWeaveVersion(url: string) {
+  const [data, setData] = useState<string | null>(null);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await fetch(url);
+        if (response.ok) {
+          const data = await response.text();
+          setData(data);
+        } else {
+          setData('');
```

Comment:
I think you want `null` here?

Reply:
default state is null so I can just remove